examples/eks-msk-opensearch/README.md

```diff
@@ -0,0 +1,113 @@
+## Multi-Tenant logging from EKS using Fluent-bit, Amazon MSK and  Amazon OpenSearch
+
+**Let us first start by explaining some of the key terms we will refer throught the solution .**
+
+* * Tenant - A tenant will be used to refer to a seperate deployment of microservices/pods in kubernetes [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+* * Broker - A broker is a software services to decouple publisher and consumers. In generic terms it will be our message queue which will store the log messages before these are sent for visulaization. 
+* * MSK - It refers to [Amazon Managed Service for Kafka](https://aws.amazon.com/msk/) ( Our broker in this example) 
+* * [Terraform](https://www.terraform.io/) - It's Hashicorp's tool to define infrastructure as a code .
```

Comment:
stray period at the end here... also some of these bullets end in periods some do not... it'd be ideal to be consistent.